bindings/python/py_src/safetensors/paddle.py

```diff
@@ -71,13 +71,21 @@ def save_file(
     return numpy.save_file(np_tensors, filename, metadata=metadata)
 
 
-def load(data: bytes, device: str = "cpu") -> Dict[str, paddle.Tensor]:
+def load(data: bytes, device: str = None) -> Dict[str, paddle.Tensor]:
     """
     Loads a safetensors file into paddle format from pure bytes.
 
     Args:
         data (`bytes`):
             The content of a safetensors file
+        device (`Dict[str, any]`, *optional*, defaults to `cpu`):
```

Comment:
```suggestion
        device (`Dict[str, any]`, *optional*, defaults to `"cpu"`):
```
Also the type is inconsistent with what is above.